src/index.js

```diff
@@ -1,50 +1,33 @@
 // import _ from 'lodash';
 import './style.css';
 
-let countStart = 0;
-const arrayList = [
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'complete To Do list project',
-    completed: false,
-    index: 1,
-  },
-];
-
+const arrayList = [];
 const getPrincial = document.getElementById('list_row');
 
-arrayList.forEach((elem) => {
-  const dataStruct = `
-    <div class="list">
-      <input type="checkbox">
-      <p class="editable">${elem.description}</p>
-    </div>
-    <div class="point" onclick="displayMenu()">
-      <div class="points"></div>
-      <div class="points"></div>
-      <div class="points"></div>
-    </div>
-    <div class="delete_img">
-      <div class="pointv"></div>
-      <div class="pointv"></div>
-      <div class="pointv"></div>
-    </div>
-  `;
-  const createDiv = document.createElement('div');
-  createDiv.classList.add('rows');
-  createDiv.classList.add(elem.index);
-  createDiv.innerHTML = dataStruct;
-  getPrincial.appendChild(createDiv);
+const editElement = ((checkEvent) => {
+  const storeData = document.getElementsByClassName(checkEvent)[0];
+  storeData.getElementsByClassName('point')[0].style.display = 'none';
+  storeData.getElementsByClassName('delete_img')[0].style.display = 'flex';
+  storeData.getElementsByClassName('editable')[0].contentEditable = true;
+  storeData.getElementsByClassName('editable')[0].focus();
+  storeData.style.backgroundColor = 'rgb(231, 230, 177)';
+});
+
+const deleteElement = ((checkEvent) => {
+  let keyCode = checkEvent.split(' ')[1];
+  keyCode -= 1;
+  arrayList.splice(keyCode, 1);
+  document.getElementsByClassName(checkEvent)[0].remove();
+  localStorage.setItem('List', JSON.stringify(arrayList));
```

Comment:
- [ ] KIndly make sure your data is preserved when the page is refreched. Kindly follow this screencast

https://user-images.githubusercontent.com/65068771/184257481-63549984-f962-465c-8817-0bd1ad5aa4c6.mp4

Reply:
Thank you now it is loaded the data from local storage